{{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/records/templates/semantic-ui/records/search.html

```diff
@@ -3,8 +3,6 @@
 {%- extends "semantic-ui/invenio_search_ui/search.html" %}
 
 {% block javascript %}
-  <!-- Because loading order of overridables -->
   {{ webpack['{%- endraw -%}{{cookiecutter.project_shortname}}{%- raw -%}-search-app.js'] }}
-  {{ super() }}
```

Comment:
Question:  Are you sure about this one?

Reply:
If it stays, the searchbar is not being displayed. 
![image](https://user-images.githubusercontent.com/15194802/101650971-d6230480-3a3c-11eb-9b76-4dd799c1e22a.png)